locale/en.po

```diff
@@ -4283,6 +4300,18 @@ msgstr ""
 "To leave the colony entirely you can click on your own cell or press the "
 "unbind all key."
 
+#: ../src/tutorial/microbe_stage/MicrobeTutorialGUI.tscn:421
+msgid "MICROBE_STAGE_HELP_MENU_AND_ZOOM"
+msgstr ""
+"If you are still struggling with game mechanics, check the help menu. You "
+"can open it with the question mark button in the bottom left.\n"
```

Comment:
```suggestion
"can open it with the question mark button in the bottom left or by pressing [thrive:input]help[/thrive:input].\n"
```
Not tested but if this works, why not add that?

Reply:
It doesn't actually work. I think we have an open issue.

Reply:
Found it: https://github.com/Revolutionary-Games/Thrive/issues/2462